src/frontends/pytorch/src/utils.cpp

```diff
@@ -459,6 +459,106 @@ Any simplified_type_interpret(Any type) {
     return type;
 }
 
+namespace {
+std::unordered_map<size_t, element::Type> bit_to_float{
+    {16, element::f16},
+    {32, element::f32},
+    {64, element::f64},
+};
+std::unordered_map<size_t, element::Type> bit_to_int{
+    // {4, element::i4}, torch don't have int4
+    {8, element::i8},
+    {16, element::i16},
+    {32, element::i32},
+    {64, element::i64},
+};
+}  // namespace
+
+void align_eltwise_input_types(const NodeContext& context,
+                               ov::Output<ov::Node>* lhs,
+                               ov::Output<ov::Node>* rhs,
```

Comment:
Let us pass them by reference instead. Simple pointer is not good here.

Reply:
There is pros and cons in refs and ptrs, is there a strong reason why ptr is bad? I think it makes sense from the readability side, you always know when value will be changed inside the function when you call it

Reply:
Someone can pass nullptr and there are no checks on it.
Using pointers will lead to extra checks to have clean code.

Reply:
Hmm, you may be right, okay I will change to refs